src/browser/api/webcontents.ts

```diff
@@ -0,0 +1,56 @@
+import { WebContents } from 'electron';
+import * as url from 'url';
+
+
+export function executeJavascript(webContents: WebContents, code: string, callback: (e: any, result: any) => void): void {
+    webContents.executeJavaScript(code, true, (result) => {
```

Comment:
can/should we do `await` here, [electron docs](https://github.com/openfin/runtime/blob/d648b1832101766bc369c7cb6b42920b95886c8e/docs/api/web-contents.md#contentsexecutejavascriptcode-usergesture-callback)?

Reply:
Rather do it in a follow up, keep this closer to current code